ADDING_NEW_CLIENTS.md

```diff
@@ -73,21 +88,20 @@ Notes/reminders:
 
 ### Workflow
 
-New services should be created in a branch based on `master`.  The branch name should include "alpha".  For example, while developing `gcloud-java-pubsub`, all Pub/Sub related work should be done in `pubsub-alpha`.  All code should be submitted through pull requests from a branch on a forked repository.  Limiting pull request size is very helpful for reviewers.  All code that is merged into the branch should be standalone and well-tested.  Any todo comments in the code should have an associated Github issue number for tracking purposes.  You should periodically pull updates from the master branch, especially if there are project-wide updates or if relevant changes have been made to the core utilities library, `gcloud-java-core`.
+New services should be created in a branch based on `master`.  The branch name should include the suffix "-alpha".  For example, while developing `gcloud-java-pubsub`, all Pub/Sub related work should be done in `pubsub-alpha`.  All code should be submitted through pull requests from a branch on a forked repository.  Limiting pull request size is very helpful for reviewers.  All code that is merged into the branch should be standalone and well-tested.  Any todo comments in the code should have an associated Github issue number for tracking purposes.  You should periodically pull updates from the master branch, especially if there are project-wide updates or if relevant changes have been made to the core utilities library, `gcloud-java-core`.
```

Comment:
I'll probably leave this out because it's hard to come up with a specific recommendation (since I won't be reviewing pull requests too much anymore) and sometimes 1500 lines of pure code is way too much whereas 1500 lines of boilerplate code (including licences, etc) is totally fine.